pkgs/development/python-modules/hledger-utils/default.nix

```
@@ -0,0 +1,48 @@
+{lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, setuptools-scm
+, rich
+, pandas
+, scipy
+, psutil
+, matplotlib
+, drawilleplot
+, asteval
+}:
+
+buildPythonPackage rec {
+  pname = "hledger-utils";
+  version = "1.12.1";
+
+  src = fetchPypi {
+    # pname = "hledger_utils";
+    inherit version pname;
+    hash = "sha256-+v627zjwy5t5MUjQJioNkRLrcOgI9LxAdscvarOikHw=";
+  };
+
+  format = "pyproject";
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  propagatedBuildInputs = [
+    setuptools
+    setuptools-scm
```

```suggestion
  nativeBuildInputs = [
    setuptools
    setuptools-scm
  ];

  propagatedBuildInputs = [
```

Those two are not runtime dependencies and should not be propagated.